doc/pod-creation-example.md

```diff
@@ -0,0 +1,88 @@
+# Example - Allocating MAC Addresses to a Pod
+
+## Create a network-attachment-definition:
+
+This example will use [ovs-cni](https://github.com/kubevirt/ovs-cni/).
+
+```yaml
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: ovs-conf
+  annotations:
+    k8s.v1.cni.cncf.io/resourceName: ovs-cni.network.kubevirt.io/br1
+spec:
+  config: '{
+      "cniVersion": "0.3.1",
+      "name": "ovs-conf",
+      "plugins" : [
+        {
+          "type": "ovs",
+          "bridge": "br1",
+          "vlan": 100
+        },
+        {
+          "type": "tuning"
+        }
+      ]
+    }'
+```
+
+**note** the project supports only json configuration for `k8s.v1.cni.cncf.io/networks`, network list will be ignored
+
+## Allocate a MAC Addresses to a pod
```

Comment:
Since Pods are mentioned with capital, maybe better to stay consistent
```suggestion
## Allocate a MAC Addresses to a Pod
```

Reply:
changed for the entire page. Done